common/djangoapps/student/views.py

```diff
@@ -756,10 +763,14 @@ def dashboard(request):
     else:
         redirect_message = ''
 
+    courses = get_courses(user)
```

Comment:
[LP-82](https://philanthropyu.atlassian.net/browse/LP-82) changed student dashboard to show recommended courses from all available courses instead of showing enrolled courses. But they have changed the mock up to remove that.